sunpy/data/data_manager/downloader.py

```diff
@@ -48,6 +48,8 @@ def download(self, url, path):
         directory = path.parent
         downloader.enqueue_file(url, directory, filename)
         try:
-            downloader.download()
+            output = downloader.download()
         except Exception as e:
             raise DownloaderError from e
+        if output.errors:
+            raise DownloaderError(output.errors[0].exception)
```

Comment:
Why only the first one?

Reply:
This function should only acquire one file, the return of errors is a list, so I want to escape it here.